### ISSUE TITLE
No collision for dropped items for a while

### ISSUE THREAD

**Issue by TamasB14** [studio-minus/ppg-bugs-and-requests#515]
**Feature description**
After humans dies wielding a weapon, most of the time the weapon breaks their bones because it's stuck.
By making the items no collide as long as it's inside a bodypart when dropped, it could solve the trick.
The item could only collide again with the body after it's taken out of the bodypart's area.

**Feature purpose**
Could solve the annoying weapon stuck into the human causing random bonebreaks when killing someone.

**Additional context**
![collision](https://user-images.githubusercontent.com/26002677/142218213-d36e2544-b435-4b25-81ae-9b7e29273e3e.png)

**Comment by mestiez** [comment]
yes I've thought about implementing this for a long time but never got to it. it should definitely be in the game.
